spec/pdc/v1/content_delivery_repo_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+
+describe PDC::V1::ContentDeliveryRepo do
+  before do
+    VCR.insert_cassette fixture_name
+  end
+
+  after do
+    VCR.eject_cassette
+  end
+
+  let(:content_delivery_repos) { PDC::V1::ContentDeliveryRepo.where(release_id: 'ceph-2.1-updates@rhel-7', service: 'pulp') }
```

Comment:
Since this isn't test setup but an actual test, could you please move PDC::V1::ContentDeliveryRepo.where.. inside the actual test?

Please consider also adding the example above as a test